src/server/courseInstance/courseInstance.service.ts

```diff
@@ -135,7 +135,13 @@ export class CourseInstanceService {
       .addOrderBy('course."numberInteger"', 'ASC')
       .addOrderBy('course."numberAlphabetical"', 'ASC')
       .addOrderBy('fall_instructors."instructorOrder"', 'ASC')
-      .addOrderBy('spring_instructors."instructorOrder"', 'ASC');
+      .addOrderBy('spring_instructors."instructorOrder"', 'ASC')
+      .addOrderBy('fall_meetings.day', 'ASC')
+      .addOrderBy('fall_meetings.startTime', 'ASC')
+      .addOrderBy('fall_meetings.endTime', 'ASC')
+      .addOrderBy('spring_meetings.day', 'ASC')
+      .addOrderBy('spring_meetings.startTime', 'ASC')
+      .addOrderBy('spring_meetings.endTime', 'ASC');
```

Comment:
Did you meant to commit this as part of the "sameAs" functionality?

Reply:
Yep, that was intentional. It's not _directly_ related, but it corrects a minor bug with "sameAs" courses showing meetings in a different order. Not the end of the world, but I had [tests that were failing][fail] when comparing the the instances of parents and children (e.g. CS 109A listing Wednesday then Monday but AC 209A listing Monday then Wednesday).

[fail]: https://github.com/seas-computing/course-planner/actions/runs/3743925308/jobs/6356676628

Reply:
Ah gotcha, okay. I wondered if it was something like that but GH didn't really show me a lot of the code around the change (or even the commit that made the change) so it was a little hard to piece together the context behind it.